### PR TITLE
add better info about backlight and MIPI DCS/DSI

### PR DESCRIPTION
**Pull request by almindor**
Clarifies naming of the crate as well as backlight manual control requirement.

See #56 for reasons behind clarifications.

### CODE REVIEW

**Comment by almindor** [comment]
@tingox @rfuest do you think this works or should I add more info in the docs somewhere as well?

**Comment by tingox** [comment]
Two things tripped me up:
1) the display I was using doesn't show anything unless the backlight is on
2) the older crate (st7789) automatically controlled the backlight pin
neither is something that can be blamed on this crate.
But perhaps a troubleshooting hint might be helpful?
Something like: "if you can't see anything on your display, try turning on / activating the backlight"

File: README.md [hunk 1]
```diff
@@ -1,12 +1,18 @@
 # mipidsi
 
 This crate provides a generic display driver to connect to TFT displays
-that implement the [MIPI DSI](https://www.mipi.org/specifications/dsi).
+that implement the [MIPI DCS](https://www.mipi.org/specifications/display-command-set).
```
Comment (rfuest) [inline README.md:4]:
```suggestion
that implement the [MIPI Display Command Set (DCS)](https://www.mipi.org/specifications/display-command-set).
```

**Comment by rfuest** [comment]
I would move the backlight note into a new "Troubleshooting" section, which would make it stand out more than being part of the notes list.

Another useful addition to the README would be a link to the API docs, which I've missed before. Perhaps in the form of some badges to docs.rs and crates.io.

**Comment by almindor** [comment]
> I would move the backlight note into a new "Troubleshooting" section, which would make it stand out more than being part of the notes list.
> 
> Another useful addition to the README would be a link to the API docs, which I've missed before. Perhaps in the form of some badges to docs.rs and crates.io.

Good calls, added badges and troubleshooting section.

File: README.md [hunk 2]
```diff
@@ -1,12 +1,22 @@
 # mipidsi
 
+<p align="center">
+    <a href="https://crates.io/crates/mipidsi"><img src="https://img.shields.io/crates/v/mipidsi.svg" alt="Crates.io"></a>
+    <a href="https://docs.rs/mipidsi"><img src="https://docs.rs/mipidsi/badge.svg" alt="Docs.rs"></a>
+    <a href="https://matrix.to/#/#rust-embedded-graphics:matrix.org"><img src="https://img.shields.io/matrix/rust-embedded-graphics:matrix.org" alt="Discuss driver on embedded-graphics on Matrix"></a>
+</p>V
```
Comment (rfuest) [inline README.md:7]:
```suggestion
</p>
```

**Comment by rfuest** [comment]
Do you also want to update `lib.rs` in this PR or open another one? IMO the docs should also contain a link to the troubleshooting doc.

**Comment by almindor** [comment]
> Do you also want to update `lib.rs` in this PR or open another one? IMO the docs should also contain a link to the troubleshooting doc.

should be done now, the troubleshooting link from docs.rs will only work once merged of course

**Pull request merged**